hbase-server/src/main/java/org/apache/hadoop/hbase/util/RegionMover.java

```diff
@@ -432,6 +434,11 @@ public boolean unload() throws InterruptedException, ExecutionException, Timeout
         }
         // Remove RS present in the exclude file
         stripExcludes(regionServers);
+
+        // Remove decommissioned RS
+        Set<ServerName> decommissionedRS = new HashSet<>(admin.listDecommissionedRegionServers());
+        regionServers.removeIf(decommissionedRS::contains);
```

Comment:
Add debug logging at least of set of decommissioned RS found?

Reply:
Sure thing